docs/usage.md

```diff
@@ -1,103 +1,480 @@
 
-* Check the `../my-tf-infra` Terraform directory against the `examples/aws/ec2_t2_only.rego` rule:
+Regula operates on Terraform HCL code and JSON plans.
 
-        ./bin/regula -d lib -d examples/aws/ec2_t2_only.rego ../my-tf-infra
+You can pass in an HCL file or a directory containing an HCL file:
 
-* Check the `../test_infra/cfn/cfntest1.yaml` CloudFormation template against a directory of custom rules:
+```
+regula run my_tf_infra/main.tf
+```
 
-        ./bin/regula -d lib -d ../custom-rules ../test_infra/cfn/cfntest1.yaml
+```
+regula run my_tf_infra
+```
 
+Because Regula does not run a `terraform plan` on the user's behalf, Regula itself does not require any credentials.
 
-It is also possible to set the name of the `terraform` executable, which is useful if you have several versions of Terraform installed:
+If you'd like to evaluate a Terraform plan, simply generate a JSON plan and pass it into Regula:
 
-    env TERRAFORM=terraform-v0.12.18 ./bin/regula -d lib/ -d rules/ ../regula-ci-example/infra_tf
+```
+terraform init
+terraform plan -refresh=false -out=plan.tfplan
+terraform show -json plan.tfplan >plan.json
+regula run plan.json
+```
 
-Note that Regula requires [Terraform 0.12+](https://www.terraform.io/downloads.html) in order to generate the JSON-formatted plan.
+Using stdin:
 
-## Running Regula locally on Windows
+```
+terraform init
+terraform plan -refresh=false -out=plan.tfplan
+terraform show -json plan.tfplan | regula run
+```
 
-Because Regula uses a bash script to automatically generate a plan, convert it to JSON, and run the Rego validations, Windows users can instead manually run the steps that Regula performs. See those steps [here](development/rule-development.md#locally-producing-a-report-on-windows).  Alternatively, you can run the script using [WSL](https://docs.microsoft.com/en-us/windows/wsl/about).
+!!! note
+    Regula can only evaluate Terraform modules that are available locally. If your Terraform configuration depends on external modules (for example from the Terraform module registry or GitHub) and you want to evaluate resources from those modules, run `terraform init` before running Regula. `terraform init` will download all external modules to a `.terraform` directory and Regula will be able to resolve them.
+
+#### CloudFormation input
+
+Regula operates on CloudFormation templates formatted as JSON or YAML, including templates generated from the AWS CDK.
+
+### Flag values
+
+`-f, --format FORMAT` values:
+
+- `json` -- A JSON report containing rule results and a summary (default)
+- `table` -- An ASCII table of rule results
+- `junit` -- The JUnit XML format
+- `tap` -- The Test Anything Protocol format
+- `none` -- Do not print any output on stdout
+
+`-t, --input type INPUT-TYPE` values:
+
+- `auto` -- Automatically determine input types (default)
+- `tf-plan` -- Terraform plan JSON
+- `cfn` -- CloudFormation template in YAML or JSON format
+- `tf` -- Terraform directory or file
```

Comment:
This is specifically "Terraform HCL directory or file" right @becki-at-luminal

Reply:
That would be correct, but I think the "HCL" is understood, and this was pulled directly from `regula run -h`, so I don't think it's necessary to add it unless you feel strongly about it.